pkg/controller/v1alpha1/localmodel/controller.go

```diff
@@ -201,9 +201,10 @@ func (c *LocalModelReconciler) ReconcileForIsvcs(ctx context.Context, localModel
 	}
 
 	for namespace := range namespaces {
+		pvcName := localModel.Name + "-" + localModel.Spec.NodeGroups[0]
```

Comment:
Later on when we support multiple node groups, the node group name needs to be retrieved from the inference service node group annotation